src/pbt/prophecy_build_tool.py

```diff
@@ -221,6 +233,16 @@ def deploy(self):
                 job_definition = json.load(_in)
 
             components = job_definition["components"]
+            fabric_id = job_definition["fabric_id"]
+
+            # if --fabric is passed, then only deploy jobs
+            if self.fabric_ids:
```

Comment:
Why? If fabric IDs aren't passed we should deploy all jobs right? To make it backward compat

Reply:
yes, that is what we are doing here,
if fabric_ids is not empty, then we are Skipping otherwise, everything goes as is.